projects/symbiosis-finance/index.js

```diff
@@ -136,21 +136,21 @@ async function tvl(chainName, timestamp, block, chainBlocks) {
     })
   })
 
-  const poolBalances = (await Promise.all(poolBalancePromises))
+  const poolBalances = await Promise.all(poolBalancePromises)
+  const allBalances = poolBalances
```

Comment:
suggestion:  you can use `sdk.util.sumSingleBalance(balances,...` here

Reply:
thank you very much! I followed your recommendation.